education/__manifest__.py

```diff
@@ -21,6 +21,7 @@
         'partner_firstname',
         'partner_second_lastname',
         'partner_contact_birthdate',
+        'partner_contact_nationality'
```

Comment:
We got around this in medical by creating an `_agpl` module that binds in the AGPL assets - https://github.com/LasLabs/vertical-medical/tree/release/10.0/medical_agpl